float64 easting
float64 northing
---
float64 easting_1
float64 northing_1
float64 easting_2
float64 northing_2
int32 expected_velocity
int32 maximal_velocity
int32 num_lanes
int32 road_width
string road_type
bool pedestrian_crossing
float64 crossing_easting
float64 crossing_northing